internal/services/order/order.go

```diff
@@ -1,13 +1,219 @@
 package order
 
-func BuyOrder(BReq BuyRequest) (bRes BuyResponse, err error) {
-	return bRes, err
+import (
+	"Tradeasy/config"
+	"Tradeasy/internal/model"
+	"Tradeasy/internal/services/stock_exchange"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type result struct {
```

Comment:
can this be renamed to be more descriptive on what is use case.

Reply:
Changed.